### PR TITLE
feat: include paths for resolving imports.

### PR DESCRIPTION
**Pull request by vinayakpatil**

### CODE REVIEW

File: examples/include-paths/main.scss [hunk 1]
```diff
@@ -0,0 +1,2 @@
+@import 'simple/main';
+@import 'node-sass/test/fixtures/custom-functions/setter';
```
Comment (DovydasNavickas) [inline examples/include-paths/main.scss]: New line :)

File: src/bundler.ts [hunk 2]
```diff
@@ -221,6 +207,29 @@ export class Bundler {
...
+            const underscoredBasename = path.basename(importData.fullPath);
+            const underscoredFilePath = path.join(underscoredDirname, `_${underscoredBasename}`);
+            try {
+                await fs.access(underscoredFilePath);
+                importData.fullPath = underscoredFilePath;
+                importData.found = true;
+            } catch (underscoreErr) {
+                for (let i = 0; i < includePaths.length; i++) {
+                    importData.fullPath = path.resolve(includePaths[i], importData.path);
+                    return this.resolveImport(importData, includePaths.slice(i + 1));
```
Comment (DovydasNavickas) [inline src/bundler.ts]: As you're always returning during the first iteration, `for` loop is effectively an `if` statement here.
Therefore, I'd rather we write it as an `if`:
```
// If there are any includePaths
if (includePaths.length != 0) {
    // Resolve fullPath using its first entry
    importData.fullPath = path.resolve(includePaths[0], importData.path);
    // Try resolving import with the remaining includePaths
    const remainingIncludePaths = includePaths.slice(1);
    return this.resolveImport(importData, remainingIncludePaths);
}
```

**Comment by DovydasNavickas** [comment]
I think we can bump the version to `2.1` and release it as such, because the `beta7` part is already tested quite thoroughly.

**Comment by DovydasNavickas** [comment]
And this is new functionality, therefore minor version bump is appropriate 🙂

**Comment by vinayakpatil** [comment]
okay, cool. Should I squash commits into one?

**Comment by DovydasNavickas** [comment]
That'd be great :) I'll merge right away.

**Comment by vinayakpatil** [comment]
Done, thanks!

**Comment by vinayakpatil** [comment]
@DovydasNavickas Can you please publish a new release? And thanks for this handy project.

**Comment by DovydasNavickas** [comment]
Trying at the moment... And getting some weird errors from npm 😄

**Comment by DovydasNavickas** [comment]
My npm gone mad at all. Thankfully, @MartynasZilinskas has successfully published, so it's out there. Thanks for PR!

**Comment by vinayakpatil** [comment]
Thank you!

**Pull request merged**